lib/components/user/monitored-trip/trip-monitored-days.tsx

```diff
@@ -0,0 +1,70 @@
+import { FormattedList, FormattedMessage, useIntl } from 'react-intl'
+
+import InvisibleA11yLabel from '../../util/invisible-a11y-label'
+import React from 'react'
+
+import { ALL_DAYS } from '../../../util/monitored-trip'
+import FormattedDayOfWeek from '../../util/formatted-day-of-week'
+import FormattedDayOfWeekCompact from '../../util/formatted-day-of-week-compact'
+import getBaseColor from '../../util/base-color'
+
+import styled from 'styled-components'
+
+interface Props {
+  days: string[]
+}
+
+const DayCircleContainer = styled.div`
+  display: flex;
+  gap: 4px;
+`
+
+const MonitoredDay = styled.span<{ baseColor: string; monitored: boolean }>`
+  align-items: center;
+  background-color: ${(props) =>
+    props.monitored ? props.baseColor : 'transparent'};
+  border: 1px solid ${(props) => (props.monitored ? props.baseColor : '#333')};
+  border-radius: 50%;
+  color: ${(props) => (props.monitored ? 'white' : 'inherit')};
+  display: flex;
+  height: 27px;
+  justify-content: center;
```

Comment:
Can you also add a `text-align: center`? It's not quite aligned as is

Reply:
`text-align: center` not causing an impact here. Is it the vertical alignment you're looking at?

Reply:
grr then yeah let's try to get it centered vertically and horizontally

Reply:
It's worse now. Should we consider forcing a font with a better bounding box?

<img width="319" alt="Screenshot 2024-04-18 at 11 35 21 AM" src="https://github.com/opentripplanner/otp-react-redux/assets/86619099/bb69093c-0923-4c0c-860d-da9c627a1e8c">

Reply:
Threw some flexbox at it and i think it looks a little better across browsers, let me know what you think!